aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -15,6 +15,8 @@
 #include <magma_types.h>
 #endif
 
+#include <stdio.h>
```

Comment:
is this include intentional?

Reply:
Oh no. Sorry about that, I used it for debugging purposes. Will remove this in the commit after your code review. Thank you for pointing it out.